src/main/java/org/opentripplanner/standalone/OTPMain.java

```diff
@@ -164,14 +165,15 @@ public void run() {
             while (true) { // Loop to restart server on uncaught fatal exceptions.
                 try {
                     grizzlyServer.run();
-                    return;
+                    return true;
```

Comment:
With your new return statement at the bottom of the method, is this return statement needed? It looks like it'll now fall through to that final `return true`.

Reply:
It is needed. We are in a `while(true)` loop.

Reply:
Oh right, it's in `if (params.server) ... while (true)`. Thanks for the clarification, I didn't look enough at the context.